test/simperium/channel_test.js

```diff
@@ -310,6 +310,19 @@ describe( 'Channel', function() {
 			} );
 		} );
 
+		it( 'should have local changes on send', function( done ) {
+			channel.once( 'send', function() {
+				bucket.hasLocalChanges( hasChanges => {
```

Comment:
Given that the callback is changed to have the error is the first argument this will become:

```
bucket.hasLocalChanges( ( error, hasChanges ) => {
} );
```

We should investigate migrating to replacing all callbacks with Promises and then using async/await. Keeping this callback pattern of the Error as the first argument will help with that migration in the future.

Reply:
OK, it will always be `null` for this method. It is now passed along as of dedf579